src/SizeEstimation.php

```diff
@@ -193,25 +193,39 @@ public static function estimateInputFromScripts(ScriptInterface $script, ScriptI
      * @return array
      */
     public static function estimateUtxo(UTXO $utxo) {
+        return self::estimateUtxoFromScripts($utxo->scriptPubKey, $utxo->redeemScript, $utxo->witnessScript);
+    }
+
+    /**
+     * @param ScriptInterface $scriptPubKey
+     * @param ScriptInterface $redeemScript
+     * @param ScriptInterface $witnessScript
+     * @return array
+     */
+    public static function estimateUtxoFromScripts(
```

Comment:
This is now the most useful function in this class, and I'd almost deprecate `SizeEstimation::estimateUtxo` for it!